synfig-studio/src/gui/canvasview.cpp

```diff
@@ -520,7 +521,7 @@ CanvasView::CanvasView(etl::loose_handle<Instance> instance,etl::handle<CanvasIn
 	statusbar                (manage(new class Gtk::Statusbar())),
 	progressbar              (manage(new class Gtk::ProgressBar())),
 	toggleducksdial          (Gtk::IconSize::from_name("synfig-small_icon_16x16")),
-	resolutiondial           (Gtk::IconSize::from_name("synfig-small_icon_16x16")),
+	resolutiondial           (new studio::ResolutionDial()),
```

Comment:
Where is the paired `delete` for this `new`?

Reply:
Done